OMEdit/OMEditGUI/Animation/AbstractAnimationWindow.cpp

```diff
@@ -207,9 +214,131 @@ void AbstractAnimationWindow::createActions()
   mpRotateCameraRightAction = new QAction(QIcon(":/Resources/icons/rotateCameraRight.svg"), tr("Rotate Right"), this);
   mpRotateCameraRightAction->setStatusTip(tr("Rotates the camera right"));
   connect(mpRotateCameraRightAction, SIGNAL(triggered()), this, SLOT(rotateCameraRight()));
+  //interactive control action
+  mpInteractiveControlAction = mpAnimationParameterDockerWidget->toggleViewAction();
+  mpInteractiveControlAction->setIcon(QIcon(":/Resources/icons/control-panel.svg"));
+  mpInteractiveControlAction->setText(tr("interactive control"));
+  mpInteractiveControlAction->setStatusTip(tr("Opens the interactive control panel"));
+  connect(mpInteractiveControlAction, SIGNAL(triggered()), this, SLOT(initInteractiveControlPanel()));
```

Comment:
I think connecting `initInteractiveControlPanel` with `mpInteractiveControlAction` is not a good idea. This means we are calling `initInteractiveControlPanel` function every time user clicks on interactive control button. Just call it when FMU file is loaded.